pkgs/development/python-modules/openrazer/daemon.nix

```diff
@@ -0,0 +1,52 @@
+{ buildPythonApplication
+, daemonize
+, dbus-python
+, fetchFromGitHub
+, fetchpatch
+, gobject-introspection
+, gtk3
+, makeWrapper
+, pygobject3
+, pyudev
+, setproctitle
+, stdenv
+, wrapGAppsHook
+}:
+
+let
+  common = import ./common.nix { inherit stdenv fetchFromGitHub; };
+in
+buildPythonApplication (common // rec {
+  pname = "openrazer_daemon";
+
+  sourceRoot = "source/daemon";
+
+  patches = [
+    # https://github.com/openrazer/openrazer/pull/680
+    (fetchpatch {
+      url = https://github.com/openrazer/openrazer/pull/680/commits/4d7078b6af0cf7de2e4ff0986daff73706ce7eb8.patch;
+      sha256 = "0v5bp3zw0csxr5day44gi6b38hm74hipbd7gvvain58b521br5fq";
+      stripLen = 1;
+    })
+  ];
+
+  outputs = [ "out" "man" ];
+
+  nativeBuildInputs = [ makeWrapper wrapGAppsHook ];
+
+  propagatedBuildInputs = [
+    daemonize
+    dbus-python
+    gobject-introspection
+    gtk3
+    pygobject3
+    pyudev
+    setproctitle
+  ];
+
+  postBuild = ''
+    DESTDIR="$out" PREFIX="" make install manpages
+  '';
+
+  meta.description = "An entirely open source user-space daemon that allows you to manage your Razer peripherals on GNU/Linux";
```

Comment:
The `meta` from `common` will be cleaned when you do this. You need `meta = common.meta // { description = ...; }` instead, same for the other packages